### ISSUE TITLE
Missunderstanding with packages

### ISSUE THREAD

**Issue by konstantin24121** [ibitcy/eo-locale#8]
@pret-a-porter if you want give a chanse to use eo-locale without react you must remove react and react-dom form peerDependecies

https://codesandbox.io/s/rw090rkw7o

I think we can made two files, one for react comunity and one for vanila using. Otherwise we force vanila-users install react as dependency

**Comment by pret-a-porter** [comment]
Yes, it is very good point. I think about monorepo with several npm packages.

**Comment by konstantin24121** [comment]
@eo-locale/core for native and @eo-locale/react for react integration will be good if you want a monorepo

**Comment by pret-a-porter** [comment]
Sound good 👍
